### ISSUE TITLE
Custom modes for Opus

### ISSUE THREAD

**Issue by icedream** [savonet/liquidsoap#88]
I wanted to ask if it is possible in any way to allow [Opus Custom](http://wiki.xiph.org/OpusFAQ#What_is_Opus_Custom.3F) modes with the %opus encoder?

The current Opus encoder only allows standardized modes and seemingly doesn't allow custom bitrate definitions either, so I was wondering.

**Comment by dbaelde** [comment]
Should we close this for inactivity?

**Comment by smimram** [comment]
I don't think we should close it. This feature would be nice, but requires some work. We should leave it open so that someone with time or strong incentive thinks about working on it...
